source/designers-guide/getting-started/index.md

```diff
@@ -128,19 +128,26 @@ You can find a complete guide to the template inheritance in the [Smarty 101 gui
 ## Template blocks
 The `Bare` theme's HTML structure is wrapped in Smarty's `block` elements. These block elements group the frontend components into small, isolated blocks that can be edited/overwritten individually. To add changes to the elements of the `Bare` theme, you __can't__ simply write code inside your file. To customize a block, you have to call the correct Shopware block name.
 
-There are 3 ways to interact with blocks and add your changes inside your new template file:
+ In Smarty there are 3 ways to interact with blocks and add your changes inside your new template file:
```

Comment:
I would rewrite this whole paragraph. do not mention append and prepend. just describe what happens when you use  or don't use `{$smarty.block.parent}`